esphome/components/mdns/mdns_component.cpp

```diff
@@ -44,6 +44,9 @@ void MDNSComponent::compile_records_() {
 #endif
 #ifdef USE_RP2040
     platform = "RP2040";
+#endif
+#ifdef USE_LIBRETINY
+    platform = "LibreTiny";
```

Comment:
As per the main comment above, this really should be something like:
```suggestion
#ifdef USE_RTL87XX
    platform = "RTL87XX";
#endif
#ifdef USE_BK72XX
    platform = "BK72XX";
```